source/precalculus/source/08-TE/03.ptx

```diff
@@ -12,6 +12,1000 @@
+            <li> <m>0</m> </li>
+            <li> <m>\frac{\pi}{3}</m> </li>
+            <li> <m>\frac{2\pi}{3}</m> </li>
+            <li> <m>\frac{4\pi}{3}</m> </li>
+            <li> <m>\frac{5\pi}{3}</m> </li>
+            <li> <m>2\pi</m> </li></ol>
+        </p>
+      </statement>
+      <answer>
+        <p>
+          C and D
+        </p>
+      </answer>
+    </task>  
+    </activity>
+
+    <remark>
+      <p>
+        Notice in <xref ref="factor-quadratic-form"/>, one of the equations did not provide a value of <m>\theta</m> that satisfies the equation. This can happen sometimes!
+      </p>
+    </remark>
+
+    <activity xml:id="factoring-practice">
+
+    <introduction>
+      <p>
+        Solve each of the following trigonometric equations on the interval <m>[0, 2\pi)</m>.
+      </p>
+    </introduction>
+
+    <task>
+      <statement>
+        <p>
+          <m>2\cos^2\theta+\cos\theta=0</m>
+        </p>
+      </statement>
+      <answer>
+        <p>
+          <m>\theta=\frac{\pi}{2}, \frac{2\pi}{3}, \frac{4\pi}{3},\frac{3\pi}{2}</m>
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          <m>\sin^2\theta-1=0</m>
+        </p>
+      </statement>
+      <answer>
+        <p>
+          <m>\theta=\frac{\pi}{2},\frac{3\pi}{2}</m>
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          <m>\cos^2\theta-\cos\theta=4</m>
+        </p>
+      </statement>
+      <hint>
+        <p>
+          Get everything over to one side before you factor!
+        </p>
+      </hint>
+      <answer>
+        <p>
+          <m>\theta=\pi</m>
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          <m>2\cos^2\theta-4=7\cos\theta</m>
+        </p>
+      </statement>
+      <answer>
+        <p>
+          <m>\theta=\frac{2\pi}{3}, \frac{4\pi}{3}</m>
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          <m>2\sec^2\theta-2\sec\theta-4=0</m>
+        </p>
+      </statement>
+      <answer>
+        <p>
+          <m>\theta=\frac{\pi}{3}, \pi, \frac{5\pi}{3}</m>
+        </p>
+      </answer>
+    </task>
+    </activity>
+
+    <remark>
+      <p>
+        In these last set of activities, let's look at another type of trigonometric equation you will see. These equations are multiple angle trigonometric equations. 
+      </p>
+    </remark>
+
+    <activity xml:id="multiple-angle-example">
+
+    <introduction>
+      <p>
+        Consider the equation <m>\sin(2\theta)=\frac{1}{2}</m>. 
+      </p>
+    </introduction>
+
+    <task>
+      <statement>
+        <p>
+          Although we are given <m>\sin(2\theta)=\frac{1}{2}</m>, let's first think about values of <m>\theta</m> that would satisfy <m>\sin\theta=\frac{1}{2}</m>. What values of <m>\theta</m> would satisfy the equation <m>\sin\theta=\frac{1}{2}</m> on the interval <m>[0, 2\pi)</m>?
```

Comment:
I think it might be better to use a different variable.  Say e.g. $\alpha = 2 \theta$, solve it for $\alpha$, then bring back $\theta$ at the end.